src/helpers/request.js

```diff
@@ -13,7 +13,7 @@ const request = async (path, manager, params) =>{
     };
 
     const { REACT_APP_SERVICE_BASE_URL } = process.env;
-
+    console.log('REQUEST BASE URL: ', REACT_APP_SERVICE_BASE_URL, ' PATH:', path);
```

Comment:
@ldvlgr is there a reason to keep this console.log here?

Reply:
To help with debugging issues related to missing/incorrect .env config file